lsp-methods.el

```diff
@@ -940,7 +940,16 @@ https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#co
       :annotation-function #'lsp--annotate
       :display-sort-function #'lsp--sort-completions)))
 
-;;; TODO: implement completionItem/resolve
+(defun lsp--resolve-completion (item)
+  (lsp--cur-workspace-check)
+  (lsp--send-changes lsp--cur-workspace)
+  (unless item
```

Comment:
A cleaner way of doing this would be `(cl-assert item nil "completion item is nil")` (see [`lsp--send-request-async`](https://github.com/emacs-lsp/lsp-mode/blob/364dd567d3bd3246d541314de03ec4b6b9f051e0/lsp-methods.el#L240))